released/discovery_center/mission_3488/step1/variables.tf

```diff
@@ -0,0 +1,179 @@
+  description = "List of Cloud Foundry org billing managers."
+  default     = []
+
+  # add validation to check if admins contains a list of valid email addresses
+  validation {
+    condition     = length([for email in var.cf_org_billing_managers : can(regex("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$", email))]) == length(var.cf_org_billing_managers)
+    error_message = "Please enter a valid email address for the subaccount admins."
+  }
+}
+
+variable "cf_org_auditors" {
+  type        = list(string)
+  description = "List of Cloud Foundry org auditors."
+  default     = []
+
+  # add validation to check if admins contains a list of valid email addresses
+  validation {
+    condition     = length([for email in var.cf_org_auditors : can(regex("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$", email))]) == length(var.cf_org_auditors)
+    error_message = "Please enter a valid email address for the subaccount admins."
+  }
+}
+
+variable "cf_space_managers" {
+  type        = list(string)
+  description = "List of managers for the Cloud Foundry space."
+  default     = []
+
+  # add validation to check if admins contains a list of valid email addresses
+  validation {
+    condition     = length([for email in var.cf_space_managers : can(regex("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$", email))]) == length(var.cf_space_managers)
+    error_message = "Please enter a valid email address for the subaccount admins."
+  }
+}
+
+variable "cf_space_developers" {
+  type        = list(string)
+  description = "List of developers for the Cloud Foundry space."
+  default     = []
+
+  # add validation to check if admins contains a list of valid email addresses
+  validation {
+    condition     = length([for email in var.cf_space_developers : can(regex("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$", email))]) == length(var.cf_space_developers)
+    error_message = "Please enter a valid email address for the subaccount admins."
+  }
+}
+
+variable "cf_space_auditors" {
+  type        = list(string)
+  description = "The list of Cloud Foundry space auditors."
+  default     = []
+
+  # add validation to check if admins contains a list of valid email addresses
+  validation {
+    condition     = length([for email in var.cf_space_auditors : can(regex("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$", email))]) == length(var.cf_space_auditors)
+    error_message = "Please enter a valid email address for the subaccount admins."
+  }
+}
+
+
+variable "service_plan__sac" {
+  type        = string
+  description = "Plan for the service instance of ABAP."
+  default     = "free"
+}
+
+
+variable "origin" {
+  type        = string
+  description = "The identity provider for the UAA user."
+  default     = "sap.ids"
+}
+
+variable "create_tfvars_file_for_next_stage" {
+  type        = bool
+  description = "Switch to enable the creation of the tfvars file for the next step."
+  default     = false
+}
+
+variable "sac_param_first_name" {

```

Comment:
Just FYI: For QAS, we'll need support for custom input fields to set these variables.